routes/helperMethods.ts

```diff
@@ -0,0 +1,325 @@
+//some helper methods for routing
```

Comment:
methods here are only supposed to be related to routing(e.g. returnData, errorhandler), please make one or multiple separate file for other helper methods that are in the same category.

Reply:
sounds good, done